frontend/src/Settings/Quality/Shapes/SizeLimitsShape.js

```diff
@@ -0,0 +1,7 @@
+import PropTypes from 'prop-types';
```

Comment:
I appreciate and respect that; however I'm not very familiar with the code base, the specific best practices being encouraged for its future, nor the technology it is based on. With this much uncertainty, I've been trying to "go with the flow" and keep things similar to what I'm already seeing.

If you can provide more specific guidance, I am happy to comply, otherwise I'm not sure what it is you want me to do.

Reply:
Removing the shape seems to me to make the code more error prone, as now there's zero context as to the schema of the object. Removing the shape wasn't one of the interpretations I had after reading his feedback. If anything, it sounded more like he was asking for a typescript version of this instead of adding more `js` files.

Reply:
Removing the shape is certainly the easiest solution, but my original thought was adding this shape to `QualityDefinitions` instead of a separate file.

Reply:
I have reverted all frontend changes and this will be resolved in my next push.